pkg/kubelet/apis/podresources/server_v1.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package podresources
+
+import (
+	"context"
+
+	"k8s.io/kubernetes/pkg/kubelet/metrics"
+
+	"k8s.io/kubelet/pkg/apis/podresources/v1"
+	"k8s.io/kubelet/pkg/apis/podresources/v1alpha1"
+)
+
+// podResourcesServerV1alpha1 implements PodResourcesListerServer
+type v1PodResourcesServer struct {
+	podsProvider    PodsProvider
+	devicesProvider DevicesProvider
+}
+
+// NewV1PodResourcesServer returns a PodResourcesListerServer which lists pods provided by the PodsProvider
+// with device information provided by the DevicesProvider
+func NewV1PodResourcesServer(podsProvider PodsProvider, devicesProvider DevicesProvider) v1.PodResourcesListerServer {
+	return &v1PodResourcesServer{
+		podsProvider:    podsProvider,
+		devicesProvider: devicesProvider,
+	}
+}
+
+func alphaDevicesToV1(alphaDevs []*v1alpha1.ContainerDevices) []*v1.ContainerDevices {
```

Comment:
Probably until we deprecate v1 (if there's ever a v2).
Given that removing the alpha API (which is actually the beta API), would break all the plugins that were written since 1.13, and require:
- For vendors, a code change (revendoring) and a new image
- For users a new version of their plugin

It doesn't seem like it would be the right call to remove the alpha API.

Thinking about this now though, we could probably look into having the alpha version call the v1 version and do the conversion in the alpha part of the code.
It might be a good followup PR :) !